darts/ad/scorers/kmeans_scorer.py

```diff
@@ -0,0 +1,216 @@
+"""
+KMeansScorer
+-----
+
+KMeans Scorer (k-means clustering) [1]_.
+The implementations is wrapped around `scikit-learn
+<https://scikit-learn.org/stable/modules/generated/sklearn.cluster.KMeans.html>`_.
+
+References
+----------
+.. [1] https://en.wikipedia.org/wiki/K-means_clustering
+"""
+
+from typing import Optional, Sequence
+
+import numpy as np
+from sklearn.cluster import KMeans
+
+from darts.ad.scorers.scorers import FittableAnomalyScorer
+from darts.logging import raise_if_not
+from darts.timeseries import TimeSeries
+
+
+class KMeansScorer(FittableAnomalyScorer):
+    def __init__(
+        self,
+        window: Optional[int] = None,
+        k: int = 2,
+        component_wise: bool = False,
+        diff_fn=None,
```

Comment:
Set the default here (and similar for the other scorers where this applies)
```suggestion
        diff_fn= "abs_diff",
```